packages/patternfly-4/react-inline-edit-extension/build/copyStyles.js

```diff
@@ -26,6 +25,7 @@ ast.stylesheet.rules = ast.stylesheet.rules.filter(rule => {
     case 'comment':
       return false;
     case 'font-face':
+      // eslint-disable-next-line no-case-declarations
       const fontFamilyDecl = rule.declarations.find(decl => decl.property === 'font-family');
```

Comment:
Change this `const` to a `var` or make the `case 'font-face':` into a block statement like:
```
case 'font-face': {
     const fontFamilyDecl = rule.declarations.find(decl => decl.property === 'font-family');
     return !unusedFontFamilyRegEx.test(fontFamilyDecl.value);
}
```

See https://eslint.org/docs/rules/no-case-declarations

Reply:
Updated